@here/harp-mapview/lib/ScreenProjector.ts

```diff
@@ -68,30 +62,10 @@ export class ScreenProjector {
      * @returns {THREE.Vector3} The projected vector (the parameter 'target').
      */
     projectVector(source: THREE.Vector3 | THREE.Vector4, target: THREE.Vector3): THREE.Vector3 {
-        target.x = source.x - this.m_center.x - this.m_cameraPosition.x;
-        target.y = source.y - this.m_center.y - this.m_cameraPosition.y;
-        target.z = source.z - this.m_center.z - this.m_cameraPosition.z;
-        target.applyMatrix4(this.m_projectionViewMatrix);
+        target.set(source.x, source.y, source.z).project(this.m_mapView!.camera);
```

Comment:
Minor ... It looks like now, we need only `camera` and this camera is always same, so maybe just make it `readonly m_camera` in constructor.

Removes also circular dependency between this module and MapView

Reply:
Fair point. Will be addressed.